src/app/services/actions/permissions-action-creator.ts

```diff
@@ -173,13 +170,6 @@ export function fetchScopes(scopesFetchType: ScopesFetchType = 'full') {
   };
 }
 
-export function getPermissionsScopeType(profile: IUser | null | undefined) {
-  if (profile?.profileType === ACCOUNT_TYPE.MSA) {
-    return PERMS_SCOPE.PERSONAL;
```

Comment:
I think an MSA account has a different scope... `DelegatedPersonal` 
Shouldn't we still pass this scope?

Reply:
I have seen it. This change is unnecessary. Let me close the PR and the issue with these comments.